lib/appoptics_apm/inst/rack.rb

```diff
@@ -162,7 +158,8 @@ def send_metrics(env, req, req_url, start, status)
         status = status.to_i
         error = status.between?(500,599) ? 1 : 0
         duration =(1000 * 1000 * (Time.now - start)).round(0)
-        AppOpticsAPM::Span.createHttpSpan(transaction_name(env), req_url, domain(req), duration, status, req.request_method, error) || ''
+        method = req.request_method
```

Comment:
is this just to shorten the line?

do you have a line length limit that you try to use? is `rubocop` the way it's done?

Reply:
yes.
I'm trying to stay below 120 chars per line, my editor windows are set up that way.
The standard is actually 80 chars, but there is a lot of discussion with regards to that restriction.